src/util/initialState.js

```diff
@@ -42,7 +42,7 @@ export const initialState = () => {
         contexts: []
       },
     },
-    recentlyEdited: [],
+    recentlyEdited: { _ROOT_: {} },
```

Comment:
Is this supposed to be the same as `ROOT_TOKEN`, or is this independent? Either way, please use a constant.

Reply:
Sure